zanshinsdk/client.py

```diff
@@ -275,6 +280,40 @@ def iter_organization_alerts(self, organization_id: Union[UUID, str],
                                                       page=page_number, page_size=page_size)
             yield from page.get('data', [])
 
+    def create_scan_target(self, organization_id: Union[UUID, str], kind: ScanTargetKind, name: str, **kwargs) -> Dict:
+        """
+        Create a new scan target in Zanshin for this organization
+        :param organization_id: the ID of the organization
+        :param kind: the Kind of scan target (AWS, GCP, AZURE)
+        :param name: the name of the scan target
+        :param kwargs: extra parameters for each kind of scan target
+        :param schedule: schedule in cron format
+        """
+        validate_class(kind, ScanTargetKind)
+        if kind == ScanTargetKind.AWS:
+            if kwargs['account']:
```

Comment:
You need to check if the `account` parameter was actually provided, and also if it is a string, and fail otherwise.